client/src/components/AppList/AppList.jsx

```diff
@@ -0,0 +1,21 @@
+import React, { useState } from 'react';
+
+import AppListing from '../AppListing/AppListing.jsx';
+
+import './AppList.scss';
+
+export default function AppList() {
+  const [position, setPosition] = useState(`Senior Front End Developer`);
```

Comment:
why not initialize the state as an empty string?

Reply:
Will change it to that at some point.  Just wanted a placeholder for now... is there a better way to include placeholders?

Reply:
okay got it! Posting a screenshot with placeholder is more visual. we can see what the code does